src/main/java/com/github/dockerjava/core/DockerClientImpl.java

```diff
@@ -1,5 +1,12 @@
 package com.github.dockerjava.core;
 
+import static com.google.common.base.Preconditions.checkNotNull;
```

Comment:
I'm using Intellij, it's re-arranged automatically like that by default. Please feel free to re-organize.